frontend/src/metabase/components/DateMonthYearWidget/DateMonthYearWidget.styled.tsx

```diff
@@ -29,16 +27,19 @@ export const MonthRoot = styled.div<MonthRootProps>`
   margin: 0.5rem 0;
   padding: 0.5rem 1rem;
   border-radius: 99px;
-  color: ${props => props.isSelected && color("text-white")};
-  background-color: ${props =>
-    props.isSelected && "var(--mb-color-background-brand)"};
+  ${props =>
+    props.isSelected &&
```

Comment:
now selected variant will use `*-selected` color and `:hover` will use `*-hover` color. This applies to all changes in this PR.